components/cam/bld/configure

```diff
@@ -1828,6 +1886,36 @@ $cfg_cppdefs .= " -DMAXPATCH_PFT=numpft+1 -DLSMLAT=1 -DLSMLON=1";
 # Building for perturbation growth tests
 if ($pergro eq "ON") { $cfg_cppdefs .= " -DPERGRO"; }
 
+# Building for MMF/super-parameterization
```

Comment:
I think this block had better be conditioned on if (use_MMF eq "ON"), at least for the addition to cfg_cppdefs.

Reply:
yea, Ben caught that last week. He has a fix being tested right now. The MMF code was erroneously being built for non-MMF cases, but amazingly the non-MMF tests were still BFB.

Reply:
Huh, I am late to the game. Interesting behavior,